src/teachers_gitlab/main.py

```diff
@@ -593,6 +593,56 @@ def action_clone(
         mg.clone_or_fetch(glb, project, local_path)
         mg.reset_to_commit(local_path, last_commit.id)
 
+@register_command('create-group', 'Create new group')
+def create_group(
+    glb: GitlabInstanceParameter(),
+    logger: LoggerParameter(),
+    entries: ActionEntriesParameter(),
+    group_name_template: ActionParameter(
+        'name',
+        required=False,
+        metavar='NAME_OF_THE_GROUP',
+        help='String name of the created group, formatted from CSV columns.'
+    ),
+    path_name_template: ActionParameter(
+        'path',
+        required=True,
+        metavar='NAME_OF_THE_PATH_TO_GROUP',
+        help='String name of the path to the created group, formatted from CSV columns.'
+    ),
+    parent_group_template: ActionParameter(
+        'from',
+        required=True,
+        metavar='REPO_PATH_WITH_FORMAT',
+        help='Parent repository path, formatted from CSV columns.'
+    ),
+):
+    for entry in entries.as_items():
+        from_group = mg.get_canonical_group(glb, parent_group_template.format(**entry))
+
+        path_name = path_name_template.format(**entry)
+        group_path = from_group.full_path + '/' + path_name
+
+        if group_name_template != None:
```

Comment:
I think that plain `if group_name_template` would be sufficient here (and, please, use `is not None` as recommended by PEP 8).